### ISSUE TITLE
Node Can't Resolve/Build Weak in Node.js LTS 6.9.4

### ISSUE THREAD

**Issue by super3** [Storj/storjshare-daemon#19]
Full log when trying to install: 

```
C:\Users\super3>npm install -g storjshare-daemon
npm WARN deprecated node-uuid@1.4.7: use uuid module instead
npm WARN deprecated jade@0.26.3: Jade has been renamed to pug, please install the latest version of pug instead of jade
npm WARN deprecated to-iso-string@0.0.2: to-iso-string has been deprecated, use @segment/to-iso-string instead.
npm WARN deprecated minimatch@0.3.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
C:\Program Files\nodejs\storjshare-restart -> C:\Program Files\nodejs\node_modules\storjshare-daemon\bin\storjshare-restart.js
C:\Program Files\nodejs\storjshare-start -> C:\Program Files\nodejs\node_modules\storjshare-daemon\bin\storjshare-start.js
C:\Program Files\nodejs\storjshare -> C:\Program Files\nodejs\node_modules\storjshare-daemon\bin\storjshare.js
C:\Program Files\nodejs\storjshare-stop -> C:\Program Files\nodejs\node_modules\storjshare-daemon\bin\storjshare-stop.js
C:\Program Files\nodejs\storjshare-destroy -> C:\Program Files\nodejs\node_modules\storjshare-daemon\bin\storjshare-destroy.js
C:\Program Files\nodejs\storjshare-logs -> C:\Program Files\nodejs\node_modules\storjshare-daemon\bin\storjshare-logs.js
C:\Program Files\nodejs\storjshare-daemon -> C:\Program Files\nodejs\node_modules\storjshare-daemon\bin\storjshare-daemon.js
C:\Program Files\nodejs\storjshare-status -> C:\Program Files\nodejs\node_modules\storjshare-daemon\bin\storjshare-status.js
C:\Program Files\nodejs\storjshare-killall -> C:\Program Files\nodejs\node_modules\storjshare-daemon\bin\storjshare-killall.js
C:\Program Files\nodejs\storjshare-create -> C:\Program Files\nodejs\node_modules\storjshare-daemon\bin\storjshare-create.js

> weak@1.0.1 install C:\Program Files\nodejs\node_modules\storjshare-daemon\node_modules\weak
> node-gyp rebuild


C:\Program Files\nodejs\node_modules\storjshare-daemon\node_modules\weak>if not defined npm_config_node_gyp (node "C:\Users\super3\AppData\Roaming\nvm\v6.9.4\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "" rebuild )
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
  weakref.cc
  win_delay_load_hook.cc
..\src\weakref.cc(17): fatal error C1083: Cannot open include file: 'stdlib.h': No such file or directory [C:\Program F
iles\nodejs\node_modules\storjshare-daemon\node_modules\weak\build\weakref.vcxproj]
C:\Program Files (x86)\Windows Kits\8.1\Include\um\winnt.h(31): fatal error C1083: Cannot open include file: 'ctype.h':
 No such file or directory (compiling source file C:\Users\super3\AppData\Roaming\nvm\v6.9.4\node_modules\npm\node_modu
les\node-gyp\src\win_delay_load_hook.cc) [C:\Program Files\nodejs\node_modules\storjshare-daemon\node_modules\weak\buil
d\weakref.vcxproj]
gyp ERR! build error
gyp ERR! stack Error: `C:\Program Files (x86)\MSBuild\14.0\bin\msbuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Users\super3\AppData\Roaming\nvm\v6.9.4\node_modules\npm\node_modules\node-gyp\lib\build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:106:13)
gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)
gyp ERR! System Windows_NT 10.0.14393
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\super3\\AppData\\Roaming\\nvm\\v6.9.4\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Program Files\nodejs\node_modules\storjshare-daemon\node_modules\weak
gyp ERR! node -v v6.9.4
gyp ERR! node-gyp -v v3.4.0
gyp ERR! not ok
C:\Program Files\nodejs
`-- storjshare-daemon@2.0.3
  +-- fd-diskspace@1.0.0  (git+https://github.com/littleskunk/fd-diskspace.git#2866a4cf147649d97f81fb33464bcbf0f8176bd5)  `-- storj-lib@6.0.12
    +-- jsen@0.6.4  (git://github.com/bugventure/jsen.git#9e57a9d653ff4ea47d364fc37da0f1659bfb8e89)
    `-- ntp-client@0.5.5  (git://github.com/bookchin/node-ntp-client.git#64a1d2318284578b90f388565de7c4b5a2441233)

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: weak@1.0.1 (node_modules\storjshare-daemon\node_modules\weak):
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: weak@1.0.1 install: `node-gyp rebuild`
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: Exit status 1```

**Comment by MeijeSibbel** [comment]
I opened a issue about this but it was closed by Gordon, as this is not a bug. I have been playing around with this error for about two days as all of my windows machines had this error. The way to fix this error is by uninstalling all the previous dependencies used by core, meaning Git, Visual studio, Phython, nodeJS... Only after removing everything and doing a clean install does it work.
By this i mean uninstalling all the `npm packages' and going to the control panel and uninstalling all the rest.

https://github.com/Storj/storjshare-daemon/issues/11
